### PR TITLE
fix(UI): [ENTESB-14025] - fix handling of ?WSDL urls for SOAP

### PR DESCRIPTION
**Pull request by kahboom**
fixes [ENTESB-14025](https://issues.redhat.com/browse/ENTESB-14025)

Also renamed the function that formats the filename to be more reflective of what it actually does..

### CODE REVIEW

**Comment by KurtStam** [comment]
@kahboom if you rebase the integration tests should turn green

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx [hunk 1]
```diff
@@ -103,9 +102,9 @@ export const ApiConnectorCreatorSelectMethod: React.FunctionComponent<IApiConnec
    */
   const onAddUrlSpecification = (e: React.FormEvent<HTMLInputElement>) => {
     const newUrl = e.currentTarget.value;
-    const fileExt = checkDocStyle(newUrl);
+    const fileExt = formatFileName(newUrl);
 
-    if (fileExt === 'wsdl' || fileExt.includes('?WSDL')) {
+    if (fileExt === 'wsdl' || fileExt.includes('?wsdl')) {
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx:107]: Wonder if doing `fileExt.toLowerCase()` would be better here, that'd cover all the bases.

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx [hunk 2]
```diff
@@ -128,9 +127,9 @@ export const ApiConnectorCreatorSelectMethod: React.FunctionComponent<IApiConnec
    * Callback for when one or more file uploads have been accepted.
    */
   const onUploadAccepted = (files: File[]): void => {
-    const fileExt = checkDocStyle(files[0].name);
+    const fileExt = formatFileName(files[0].name);
 
-    if (fileExt === 'wsdl' || fileExt.includes('?WSDL')) {
+    if (fileExt === 'wsdl' || fileExt.includes('?wsdl')) {
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx:132]: Kind of the same code in 2 places, would it be better to create a function like `isWsdlFile(filename: string)`?

**Comment by kahboom** [comment]
@dependabot rebase

**Pull request merged**